Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

PS C:\Users\Computka\Documents\GitHub\fieldOrders> & 'C:\Users\Computka\AppData\Local\Programs\Python\Python36-32\python.exe' 'c:\Users\Computka\.vscode\extensions\ms-python.python-2020.2.63072\pythonFiles\ptvsd_launcher.py' '--default' '--client' '--host' 'localhost' '--port' '61700' 'c:\Users\Computka\Documents\GitHub\fieldOrders\chart.py'
PS C:\Users\Computka\Documents\GitHub\fieldOrders> cd 'c:\Users\Computka\Documents\GitHub\fieldOrders'; ${env:PYTHONIOENCODING}='UTF-8'; ${env:PYTHONUNBUFFERED}='1'; & 'C:\Users\Computka\AppData\Local\Programs\Python\Python36-32\python.exe' 'c:\Users\Computka\.vscode\extensions\ms-python.python-2020.2.63072\pythonFiles\ptvsd_launcher.py' '--default' '--client' '--host' 'localhost' '--port' '61731' 'c:\Users\Computka\Documents\GitHub\fieldOrders\chart.py'
PS C:\Users\Computka\Documents\GitHub\fieldOrders> cd 'c:\Users\Computka\Documents\GitHub\fieldOrders'; ${env:PYTHONIOENCODING}='UTF-8'; ${env:PYTHONUNBUFFERED}='1'; & 'C:\Users\Computka\AppData\Local\Programs\Python\Python36-32\python.exe' 'c:\Users\Computka\.vscode\extensions\ms-python.python-2020.2.63072\pythonFiles\ptvsd_launcher.py' '--default' '--client' '--host' 'localhost' '--port' '61795' 'c:\Users\Computka\Documents\GitHub\fieldOrders\chart.py'
PS C:\Users\Computka\Documents\GitHub\fieldOrders> cd 'c:\Users\Computka\Documents\GitHub\fieldOrders'; ${env:PYTHONIOENCODING}='UTF-8'; ${env:PYTHONUNBUFFERED}='1'; & 'C:\Users\Computka\AppData\Local\Programs\Python\Python36-32\python.exe' 'c:\Users\Computka\.vscode\extensions\ms-python.python-2020.2.63072\pythonFiles\ptvsd_launcher.py' '--default' '--client' '--host' 'localhost' '--port' '61853' 'c:\Users\Computka\Documents\GitHub\fieldOrders\chart.py'
Help on package PIL:

NAME
    PIL - Pillow (Fork of the Python Imaging Library)

DESCRIPTION
    Pillow is the friendly PIL fork by Alex Clark and Contributors.
        https://github.com/python-pillow/Pillow/

    Pillow is forked from PIL 1.1.7.

    PIL is the Python Imaging Library by Fredrik Lundh and Contributors.
    Copyright (c) 1999 by Secret Labs AB.

    Use PIL.__version__ for this Pillow version.

    ;-)

PACKAGE CONTENTS
    BdfFontFile
    BlpImagePlugin
    BmpImagePlugin
    BufrStubImagePlugin
    ContainerIO
    CurImagePlugin
    DcxImagePlugin
    DdsImagePlugin
    EpsImagePlugin
    ExifTags
    FitsStubImagePlugin
    FliImagePlugin
    FontFile
    FpxImagePlugin
    FtexImagePlugin
    GbrImagePlugin
    GdImageFile
    GifImagePlugin
    GimpGradientFile
    GimpPaletteFile
    GribStubImagePlugin
    Hdf5StubImagePlugin
    IcnsImagePlugin
    IcoImagePlugin
    ImImagePlugin
    Image
    ImageChops
    ImageCms
    ImageColor
    ImageDraw
    ImageDraw2
    ImageEnhance
    ImageFile
    ImageFilter
    ImageFont
    ImageGrab
    ImageMath
    ImageMode
    ImageMorph
    ImageOps
    ImagePalette
    ImagePath
    ImageQt
    ImageSequence
    ImageShow
    ImageStat
    ImageTk
    ImageTransform
    ImageWin
    ImtImagePlugin
    IptcImagePlugin
    Jpeg2KImagePlugin
    JpegImagePlugin
    JpegPresets
    McIdasImagePlugin
    MicImagePlugin
    MpegImagePlugin
    MpoImagePlugin
    MspImagePlugin
    PSDraw
    PaletteFile
    PalmImagePlugin
    PcdImagePlugin
    PcfFontFile
    PcxImagePlugin
    PdfImagePlugin
    PdfParser
    PixarImagePlugin
    PngImagePlugin
    PpmImagePlugin
    PsdImagePlugin
    PyAccess
    SgiImagePlugin
    SpiderImagePlugin
    SunImagePlugin
    TarIO
    TgaImagePlugin
    TiffImagePlugin
    TiffTags
    WalImageFile
    WebPImagePlugin
    WmfImagePlugin
    XVThumbImagePlugin
    XbmImagePlugin
    XpmImagePlugin
    __main__
    _binary
    _imaging
    _imagingcms
    _imagingft
    _imagingmath
    _imagingmorph
    _imagingtk
    _tkinter_finder
    _util
    _version
    _webp
    features

CLASSES
    builtins.OSError(builtins.Exception)
        UnidentifiedImageError

    class UnidentifiedImageError(builtins.OSError)
     |  Base class for I/O related errors.
     |
     |  Method resolution order:
     |      UnidentifiedImageError
     |      builtins.OSError
     |      builtins.Exception
     |      builtins.BaseException
     |      builtins.object
     |
     |  Data descriptors defined here:
     |
     |  __weakref__
     |      list of weak references to the object (if defined)
     |
     |  ----------------------------------------------------------------------
     |  Methods inherited from builtins.OSError:
     |
     |  __init__(self, /, *args, **kwargs)
     |      Initialize self.  See help(type(self)) for accurate signature.
     |
     |  __new__(*args, **kwargs) from builtins.type
     |      Create and return a new object.  See help(type) for accurate signature.
     |
     |  __reduce__(...)
     |      helper for pickle
     |
     |  __str__(self, /)
     |      Return str(self).
     |
     |  ----------------------------------------------------------------------
     |  Data descriptors inherited from builtins.OSError:
     |
     |  characters_written
     |
     |  errno
     |      POSIX exception code
     |
     |  filename
     |      exception filename
     |
     |  filename2
     |      second exception filename
     |
     |  strerror
     |      exception strerror
     |
     |  winerror
     |      Win32 exception code
     |
     |  ----------------------------------------------------------------------
     |  Methods inherited from builtins.BaseException:
     |
     |  __delattr__(self, name, /)
     |      Implement delattr(self, name).
     |
     |  __getattribute__(self, name, /)
     |      Return getattr(self, name).
     |
     |  __repr__(self, /)
     |      Return repr(self).
     |
     |  __setattr__(self, name, value, /)
     |      Implement setattr(self, name, value).
     |
     |  __setstate__(...)
     |
     |  with_traceback(...)
     |      Exception.with_traceback(tb) --
     |      set self.__traceback__ to tb and return self.
     |
     |  ----------------------------------------------------------------------
     |  Data descriptors inherited from builtins.BaseException:
     |
     |  __cause__
     |      exception cause
     |
     |  __context__
     |      exception context
     |
     |  __dict__
     |
     |  __suppress_context__
     |
     |  __traceback__
     |
     |  args

VERSION
    7.0.0

FILE
    c:\users\computka\appdata\local\programs\python\python36-32\lib\site-packages\pil\__init__.py